Issue #16

dalSara opened the pull request:
I dont have anything more to add at the moment. Would be nice if someone else will look in to it and maybe add some of the missing info.

Commit:
first draft of info for new employes

dalSara committed:
fixed some spelling in draft

dalSara committed:
More spelling + comments

dalSara committed:
updated draft

dalSara committed:
More software and added unit4 and cards + alarm

Reasonable-Solutions committed:
Update software.md

i added some more text

dalSara committed:
Merge pull request #19 from Creuna-Oslo/Reasonable-Solutions-patch-1

Update software.md

dalSara committed:
Added info on ADO and other small things.

asbjornh commented on new-employees/Secrets-in-the-office.md:
```diff
@@ -0,0 +1,84 @@
+There is some good instruction videos in the workplace [Unit4 Group](https://creuna.facebook.com/groups/1056335684500691/videos/). The information for registrate your hours for diffrent projects is usually given by the project leader. 
+
+## Access cards and the alarm ##
+You have most likely got an access card and a code for that card. Remember that code. You will need that code if you arrive early (before 8.00)or work late(after 16.00).  After 23.00 and in the weekends the alarm is active in the office. If you use your card at the card reader the alarm will be off for 3 hours. If the alarm starts you will have to call Nokas on 02580 to deactivate it. You will ha ...
```
Should we have details about  alarm system times publicly available?

pellebjerkestrand replied:
No, we should not.

dalSara replied:
Added it on request. But now it is gone.

dalSara committed:
Removed info

asbjornh commented:
Here's a clickable thing to see the related issue: #16

asbjornh commented:
I found several typos when reading through it, but I didn't have time to comment on them. If this gets merged before I find the time, we can just create a new PR for that.

Reasonable-Solutions committed:
Update Secrets-in-the-office.md (#25)

i did typos.
The diff is going to look like a mess.

Merging this since no one else did it.

Reasonable-Solutions commented:
>  Would be nice if someone else will look in to it and maybe add some of the missing info.

We cannot see what is missing because it is not there. 

Merge this and we can  I T E R A T E.
